cluster-install/readme.adoc

```diff
@@ -82,38 +92,38 @@ Then you need to follow the instructions in https://github.com/kubernetes/kops/b
 
 By default, `create cluster` command creates a single master node and two worker nodes in the specified zones.
 
-Create a Kubernetes cluster using the following command. This will create a cluster with a single master, multi-node and multi-az configuration:
+Create a Kubernetes cluster using the following command. For the purposes of this demonstration, we will use a cluster name of example.cluster.com as our registered DNS. This will create a cluster with a single master, multi-node and multi-az configuration:
 
     kops create cluster \
-      --name cluster.kubernetes-aws.io \
-      --zones us-east-1d,us-east-1e \
-      --state s3://kubernetes-aws-io \
+      --name example.cluster.com \
+      --zones $AWS_AVAILABILITY_ZONES \
+      --state $KOPS_STATE_STORE \
```

Comment:
remove this option all together? Should not be required to set if `KOPS_STATE_STORE` is set as env variable?

Reply:
which option?

Reply:
oh, remove the repeated reference to the state store.  never mind